src/raftstore/store/fsm/store.rs

```diff
@@ -430,7 +440,7 @@ impl<'a, T: Transport, C: PdClient> StoreFsmDelegate<'a, T, C> {
         RAFT_EVENT_DURATION
             .with_label_values(&[tick.tag()])
             .observe(duration_to_sec(t.elapsed()) as f64);
-        slow_log!(t, "{} handle timeout {:?}", self.fsm.store.tag, tick);
+        slow_log!(t, "[store {}] handle timeout {:?}", self.fsm.store.id, tick);
```

Comment:
The store field won't work here. Can you workaround it? The rest is mostly fine.

Reply:
How about letting the slow_log accepts a message or a reason (in this case, it is "handle timeout") and zero or more fields? Then in the slow log implementation, the slow log message/reason is a field and these extra fields are also fields.

Reply:
This PR focus on fsm/store module, let's leave the change to another PR (I will put it on my TODO list),

Reply:
Okay